src/styles/UTILS/_normalise.scss

```diff
@@ -0,0 +1,11 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
```

Comment:
not sure if *  box-sizing: border-box; is common practice as we dont know what else will be implemented on this project

margin: 0 is style resetting, not normalizing

Reply:
Good point about `*` selector, however, not critical, but still consider using it in-place

Reply:
About styling `body` – good point, it's more about resetting, not normalizing